engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -234,6 +234,32 @@ public void testExportFromAsExport() throws Exception {
     parseTest("from prj.Data.Foo as Bar export Baz, Quux");
   }
 
+  @Test
+  public void testTextLiteral() throws Exception {
+    parseTest("""
+    main = "I'm an inline raw text!"
+    """);
+  }
+
+  @Test
+  public void testLambda() throws Exception {
+    parseTest("""
+    f = map _->alphabet
+    """);
+  }
+
+  @Test
+  public void testBench() throws Exception {
```

Comment:
I guess the real name of the test is sth like like `testMultiParameterFunction` ?

Reply:
Done in [d13cc31](https://github.com/enso-org/enso/pull/3747/commits/d13cc31dfdbb3bfa40a864baafb033a762655a5c)